server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -237,4 +315,29 @@ private void printArt() {
             + "###################################################################\n";
     System.out.println(art);
   }
+
+  public static class Builder {
```

Comment:
does the constructor for this need to be public? if not, we can make the constructor private this UnityCatalog class and just expose a public static method `builder()` that returns a new builder instance.

Reply:
this mimic are armeria builder. let just borrow the same pattern :)